account/identity.go

```diff
@@ -140,10 +142,18 @@ func (m *GormIdentityRepository) Create(ctx context.Context, model *Identity) er
 	}
 	err := m.db.Create(model).Error
 	if err != nil {
-		goa.LogError(ctx, "error adding Identity", "error", err.Error())
+		log.LogError(ctx, map[string]interface{}{
+			"identity": model,
+			"err":      err.Error(),
+		}, "Unable to create the identity")
 		return errors.WithStack(err)
```

Comment:
@kwk I was playing with the error outputs, e.g. for the `workitem_repository.go`. I found that the `errors.WithStack` or `errors.NewBadParam...` truly report the error to the client. However, the logs of the server don't inform about such an error.


Piece of code:
```
	if extendedTypeName != nil {
		extendedType := WorkItemType{}
		db := r.db.First(&extendedType, "name = ?", extendedTypeName)
		if db.RecordNotFound() {
			return nil, errors.NewBadParameterError("extendedTypeName", *extendedTypeName)
		}
``` 

Therefore, I think that logging the errors before returning them isn't a bad practice. Logs are cheap :), if they aren't duplicated. Do you agree ?